### DISCUSSION TITLE
현재 PPT로 화면구성하고 있습니다. 추후 Figma로 프로토타입 만들려고 합니다.

### DISCUSSION THREAD

**Discussion by JS-HAN214** [receiptAutoScan/receipt-backend#8] [General]
혹시 조언해 주실 부분 있으시면 많은 도움 부탁드립니다.

**Comment by ej20001000** [comment]
그게 편하시면 그렇게 하셔도 되는데 그렇게 되면 두 번 하게 되는 것 아닌가요?

**Reply by JS-HAN214** [reply]
일을 두번할 수도 있지만 프로토타입은 만들어봐야겠더라구요.
https://www.figma.com/file/3h8Z95k2XcLKzqRRDlfu6I/receipt?type=design&node-id=0%3A1&mode=design&t=1CVwMc1dBD2esnhe-1
Figma 링크입니다.

**Reply by ej20001000** [reply]
7번 8번 화면 등에서 내비게이션이 가능하게 이런게 필요하지 않을까요?

https://alikong.tistory.com/34

<img width="564" alt="image" src="https://github.com/receiptAutoScan/receipt-backend/assets/53468197/51e88dcd-0430-4fe4-a1d4-4fb3759973b9">

<img width="590" alt="image" src="https://github.com/receiptAutoScan/receipt-backend/assets/53468197/147e5c99-2696-4fda-9e42-5b0d2b232c13">

**Reply by JS-HAN214** [reply]
조언 감사해요!
